docs/tools/developer-tools/jupyter-notebooks.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Jupyter Notebooks
+description: Experiment with contracts in Jupyter Notebooks
+sidebar_label: Jupyter Notebooks
+sidebar_position: 71
+---
+
+# Jupyter Notebooks
+
+Jupyter Notebooks are a document format that can include code, text, and other rich output. And with the appropriate setup they support Soroban Rust contracts.
+
+To use Soroban contracts in a Jupyter Notebook, the following setup is required. The following setup uses Visual Studio Code, but any Jupyter client and server can be used.
+
+:::caution Rust support in Jupyter Notebooks is experimental. You might run into bugs, or unexpected behavior. :::
```

Comment:
For some reason, all the content after this is wrapped by the message box. Maybe we need something a hard linebreak?

```suggestion
:::caution 
Rust support in Jupyter Notebooks is experimental. You might run into bugs, or unexpected behavior.
:::
```